referral-form-data-process/serverless.yml

```diff
@@ -21,8 +21,10 @@ provider:
 functions:
   main:
     environment:
-      CLIENTEMAIL: ${opt:CLIENTEMAIL}
-      PRIVATEKEY: ${opt:PRIVATEKEY}
+      CLIENT_EMAIL: ${opt:CLIENT_EMAIL}
```

Comment:
Remember to also update the circle CI config for these opts. https://github.com/LBHackney-IT/social-care-referral-form-ingestion/blob/main/.circleci/config.yml#L150

Reply:
Thanks for spotting 🙌🏾

Reply:
Done in d1954c2